samples/sensor/bme680/prj.conf

```diff
@@ -0,0 +1,29 @@
+CONFIG_STDOUT_CONSOLE=y
+CONFIG_I2C=y
+CONFIG_I2C_0=y
```

Comment:
This is board-specific and shouldn't be in a generic project file. It's also why CI is failing, because frdm_kw41z enables I2C_1 when I2C is enabled.

Reply:
ok. what about `CONFIG_STDOUT_CONSOLE`